boards/nano33ble_rev2/bootloaders/README.md

```diff
@@ -0,0 +1,23 @@
+Nano 33 BLE Sense Rev2 Bootloader Binaries
+===============================
+
+This folder contains pre-built bootloader binaries for the Nano 33 BLE Sense Rev2 board.
+The file names contain information about which bootloader it is and which
+address it is compiled for.
```

Comment:
Replace the contents of this readme to just point to the nano33ble/bootloaders folder.